### PR TITLE
Support relative link

### LINKED ISSUE #44: image is not displayed after generating the pdf ?
Is possible add css and image file path ?

### PR DESCRIPTION
**Pull request by raytfer**
Support relative link for html-pdf command.

This should fix #44 and #13.

### CODE REVIEW

File: src/scripts/pdf_a4_portrait.coffee [hunk 1]
```diff
@@ -32,7 +32,11 @@ page = webpage.create()
 if options.httpHeaders
   page.customHeaders = options.httpHeaders
 
-page.content = json.html
+if options.base
+  page.setContent(json.html, options.base)
```
Comment (marcbachmann) [inline src/scripts/pdf_a4_portrait.coffee]: Nice. I didn't know of that option.

File: src/scripts/pdf_a4_portrait.coffee [hunk 2]
```diff
@@ -32,7 +32,11 @@ page = webpage.create()
...
+else
+  page.content = json.html
```
Comment (marcbachmann) [inline src/scripts/pdf_a4_portrait.coffee]: Can we also use setContent here for consistency reasons?

Comment (raytfer) [inline src/scripts/pdf_a4_portrait.coffee]: sure, see the next commit

**Comment by raytfer** [comment]
@marcbachmann any updates?

**Comment by marcbachmann** [comment]
I've squashed that and merged into master. Thanks.

**Comment by marcbachmann** [comment]
I've just published that in `html-pdf@2.0.0`

**Comment by konstantinkrassmann** [comment]
Is there a example for this setting ?

**Comment by karex** [comment]
I've came across this "relative link"-issue, and found a working solution to create the base URL in the convert script, and it looks like this:
```javascript
let path = require('path');
let currentPath = 'file:///' + path.resolve('.').replace(/\\/g, '/') + '/';
```
Then you can pass this base path to the `pdf.create` method in the options parameter like this:
```javascript
let options = {
    format: 'A4',
    base: currentPath
};
pdf.create(html, options).toFile(pdfFile, function(err, res) {
    if (err) {
        return console.log(err);
    }
    console.log(res);
});
```

**Pull request closed without merging**